statbot/client.py

```diff
@@ -322,6 +326,7 @@ def _init_sql(self, trans):
 
         with self.sql.transaction() as trans:
             self.sql.add_user(trans, member)
+            self.sql.add_member(trans, member)
```

Comment:
can we just use `upsert_member` instead?

Reply:
I don't see why. The event is `on_member_join`, which means that that `Member` can't already exist in the database anywhere. So just use an update.

However, I have changed the `add_user` to `upsert_user`, since it is logically possible for a user to already be tracked but not have joined a particular guild.

Reply:
it would be less code. the number 1 cause of bugs is people writing code

Reply:
Huh? How would it be less code?
Whether you do an insert or an upsert, that's still code.

Reply:
if you called `upsert_member` instead, you could get rid of the `add_member` code